src/client/app/components/Button/index.js

```diff
@@ -0,0 +1,3 @@
+import Button from './Button';
```

Comment:
There is really no need of such facade for a single component inside the module/dir.
_This comment applies to all similar parts of the project._